<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
    font-family: &quot;微软雅黑&quot;, &quot;Arial&quot;, sans-serif;
    color: #333333;
    font-size: 14px;
}

/* 针对所有的 QLabel */
QLabel {
    font-family: &quot;微软雅黑&quot;, &quot;Arial&quot;, sans-serif;
    color: #333333;
    font-size: 14px;
}
/* 所有控件的边框 */
* {
    border: none;
    border-style: none;
}

/* 使所有控件的边框透明 */
* {
    border-color: transparent;
}

/* 主窗口样式 */
QWidget#MainWindow {
    background-color: #222222; /* 主窗口背景色 */
}

/* 按钮样式 */
QPushButton {
    background-color: #333333; /* 按钮的背景色 */
    color: #FFFFFF; /* 按钮的文本颜色 */
    border: 2px solid #666666; /* 按钮的边框 */
    border-radius: 5px; /* 按钮的边框圆角 */
    padding: 5px 10px; /* 按钮的内边距 */
}

/* 按钮在鼠标悬停时的样式 */
QPushButton:hover {
    background-color: #555555;
}

/* 按钮在被按下时的样式 */
QPushButton:pressed {
    background-color: #222222;
}

/* 滑块样式 */
QSlider {
    background: #222222; /* 滑块的背景色 */
}

/* 滑轨样式 */
QSlider::groove:horizontal {
    border: 1px solid #666666;
    height: 8px; /* 滑轨的高度 */
    background: #111111; /* 滑轨的背景色 */
    margin: 2px 0;
    border-radius: 4px;
}

/* 滑块样式 */
QSlider::handle:horizontal {
    background: #666666; /* 滑块的背景色 */
    border: 1px solid #666666;
    width: 16px; /* 滑块的宽度 */
    height: 16px; /* 滑块的高度 */
    margin: -4px 0;
    border-radius: 8px;
}

/* 刻度线样式 */
QSlider::tickmark:horizontal {
    width: 5px; /* 刻度线的宽度 */
    height: 5px; /* 刻度线的高度 */
    background: #666666; /* 刻度线的颜色 */
    border-radius: 2px;
}

/* 刻度值样式 */
QSlider::add-page:horizontal {
    background: #666666; /* 刻度值的颜色 */
}

/* 已选取的刻度值样式 */
QSlider::sub-page:horizontal {
    background: #888888; /* 已选取的刻度值的颜色 */
}

/* QLabel样式 */
QLabel {
    color: #CCCCCC; /* 文本颜色 */
    background-color: #222222; /* 背景颜色 */
    border: 1px solid #666666; /* 边框 */
    padding: 5px; /* 内边距 */
    border-radius: 4px; /* 边框圆角 */
}
</string>
  </property>
  <widget class="QGroupBox" name="btnGroup">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>540</y>
     <width>791</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <widget class="QPushButton" name="PreMusicBtn">
      <property name="text">
       <string>上一曲</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="NextMusicBtn">
      <property name="text">
       <string>下一曲</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="PlayAndPauseMusicBtn">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="VolumeChangeBtn">
      <property name="text">
       <string>音量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="VolumeSlider">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="PlayListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#PlayListWidget::item {
    background-color: #393d49;
    color: #ffffff;
    border: transparent;
    padding: 8px;
}

#PlayListWidget::item:hover {
    background-color: #4e5465;
}

#PlayListWidget::item:selected {
    border-left: 5px solid #009688;
}
/* 滚动条样式 */
QScrollBar:vertical {
    border: none; /* 垂直滚动条的边框 */
    background: transparent; /* 垂直滚动条的背景 */
    width: 12px; /* 垂直滚动条的宽度 */
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:vertical {
    background: #666666; /* 滑块的背景色 */
    min-height: 20px; /* 滑块的最小高度 */
    border-radius: 6px; /* 滑块的边框圆角 */
}

QScrollBar::handle:vertical:hover {
    background: #888888; /* 滑块的背景色（悬停时） */
}

QScrollBar::sub-line:vertical {
    background: transparent; /* 滚动条上部按钮的背景 */
    height: 0px;
}

QScrollBar::add-line:vertical {
    background: transparent; /* 滚动条下部按钮的背景 */
    height: 0px;
}

QScrollBar::sub-page:vertical {
    background: transparent; /* 滚动条上部区域的背景 */
}

QScrollBar::add-page:vertical {
    background: transparent; /* 滚动条下部区域的背景 */
}
/* 列表控件的背景 */
QListWidget {
    background-color: #222222; /* 列表控件的背景色 */
}

/* 水平滚动条样式 */
QScrollBar:horizontal {
    border: none; /* 水平滚动条的边框 */
    background: transparent; /* 水平滚动条的背景 */
    height: 12px; /* 水平滚动条的高度 */
    margin: 0px 0px 0px 0px;
}

QScrollBar::handle:horizontal {
    background: #666666; /* 滑块的背景色 */
    min-width: 20px; /* 滑块的最小宽度 */
    border-radius: 6px; /* 滑块的边框圆角 */
}

QScrollBar::handle:horizontal:hover {
    background: #888888; /* 滑块的背景色（悬停时） */
}

QScrollBar::sub-line:horizontal {
    background: transparent; /* 滚动条左部按钮的背景 */
    width: 0px;
}

QScrollBar::add-line:horizontal {
    background: transparent; /* 滚动条右部按钮的背景 */
    width: 0px;
}

QScrollBar::sub-page:horizontal {
    background: transparent; /* 滚动条左部区域的背景 */
}

QScrollBar::add-page:horizontal {
    background: transparent; /* 滚动条右部区域的背景 */
}

</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="MediaArea">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QSlider" name="timeSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>701</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {
    background: #333333; /* 滑块的背景色 */
 	border-radius: 4px; /* 边框圆角 */
}

/* 滑轨样式 */
QSlider::groove:horizontal {
    border: 1px solid #666666;
    height: 8px; /* 滑轨的高度 */
    background: #222222; /* 滑轨的背景色 */
    margin: 2px 0;
    border-radius: 4px;
}

/* 滑块样式 */
QSlider::handle:horizontal {
    background: #888888; /* 滑块的背景色 */
    border: 1px solid #666666;
    width: 16px; /* 滑块的宽度 */
    height: 16px; /* 滑块的高度 */
    margin: -4px 0;
    border-radius: 8px;
}

/* 刻度线样式 */
QSlider::tickmark:horizontal {
    width: 5px; /* 刻度线的宽度 */
    height: 5px; /* 刻度线的高度 */
    background: #666666; /* 刻度线的颜色 */
    border-radius: 2px;
}

/* 刻度值样式 */
QSlider::add-page:horizontal {
    background: #666666; /* 刻度值的颜色 */
}

/* 已选取的刻度值样式 */
QSlider::sub-page:horizontal {
    background: #888888; /* 已选取的刻度值的颜色 */
}
</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>476</y>
      <width>95</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #CCCCCC; /* 文本颜色 */
    background-color: #333333; /* 背景颜色 */
    border: 1px solid #666666; /* 边框 */
    padding: 5px; /* 内边距 */
    border-radius: 4px; /* 边框圆角 */
}
</string>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="VideoTip">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>571</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>0</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#CloseBtn {
    background-color: #222222; /* 主窗口背景色 */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="HideBtn">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>0</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#HideBtn {
    background-color: #222222; /* 主窗口背景色 */
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
